packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+
+export default {
+  title: 'Emails/cadReminder',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
+
+const commonPropsWithOverrides = (
+  overrides: Partial<StorybookEmailArgs> = {}
+) =>
+  Object.assign({
+    template: 'cadReminder',
+    doc: 'The Connect Another Device Reminder is sent when [TODO: documentation].',
+    variables: {
+      buttonText: 'Sync device',
+      anotherDeviceURL:
+        '/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
+      iosURL:
+        'https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png',
+      androidURL:
+        'https://accounts-static.cdn.mozilla.net/product-icons/google-play.png',
+      ...overrides,
+    },
+  });
```

Comment:
I think this function needs just a little more finessing.

To start, it looks like `storybookEmail` is looking for a `description` property, but `doc` is provided instead (are these the same thing?), and if you were to manually set `description` on the individual story, like:

```js
CadReminderMobile.args = commonPropsWithOverrides({
  description: 'This is a description',
});
```

This doesn't work because the argument object `overrides` is being merged into the child `variables` object, not the top-level object.

Additionally, `commonPropsWithOverrides` is typed to receive an object with an interface of `Partial<StorybookEmailArgs>`, so effectively:

```ts
Partial<{
  template: string;
  description?: string;
  variables: Record<string, any>;
}>
```

But since the `overrides` object is being applied directly to the child `variables` object it would seem we never need to override `template` or `description`. Therefor I think instead of typing it with `Partial<StorybookEmailArgs>` I would suggest you type it as `Record<string, string>`. Or if you want to get even more specific with the types you could do [this](https://gist.github.com/jodyheavener/fab1a8072c29b2cc45953c15d27330f7), which extracts types from the default values. Either of these solutions will also allow you to remove the `/*@ts-ignore */` bits.

Reply:
I assumed that we do not want to override the template name and description of a story and was thinking to disable editable controls as well for them, something like [this](https://storybook.js.org/docs/riot/essentials/controls#disable-controls-for-specific-properties) maybe?

Reply:
Yup! We can still update the types for this